coda/coda_replication/views.py

```diff
@@ -434,6 +432,6 @@ def queue_list(request):
         return HttpResponseBadRequest('Page does not exist.')
 
     feedText = etree.tostring(atomFeed, pretty_print=True)
-    feedText = '<?xml version="1.0"?>\n{0}'.format(feedText)
+    feedText = b'<?xml version="1.0"?>\n %s' % feedText
```

Comment:
We should replace `%s` with `%b` to avoid any confusion.

Reply:
In fact, that should be done in a few other places here too. Currently `%s` _is_ an alias for `%b`, but we should be using `%b` for clarity, as we're making a bytes object, not a str.

Reply:
I will make these changes.